src/main/java/net/neoforged/neoforge/network/registration/NetworkRegistry.java

```diff
@@ -78,17 +78,11 @@
  * Additionally, this registry is responsible for handling all packets that are not natively known once they arrive at the receiving end.
  * <p>
  * To prevent payloads from being send to a client that has no idea what to do with them, the registry provides endpoints for the vanilla code base to check if a packet can be send to a client.
- * </p>
  */
 @ApiStatus.Internal
 public class NetworkRegistry {
     private static final Logger LOGGER = LogUtils.getLogger();
 
-    private static final AttributeKey<NetworkPayloadSetup> ATTRIBUTE_PAYLOAD_SETUP = AttributeKey.valueOf("neoforge:payload_setup");
-    private static final AttributeKey<Set<ResourceLocation>> ATTRIBUTE_ADHOC_CHANNELS = AttributeKey.valueOf("neoforge:adhoc_channels");
-    private static final AttributeKey<ConnectionType> ATTRIBUTE_CONNECTION_TYPE = AttributeKey.valueOf("neoforge:connection_type");
-    private static final AttributeKey<PacketFlow> ATTRIBUTE_FLOW = AttributeKey.valueOf("neoforge:flow");
```

Comment:
Why did you remove this channel attribute (and not mention it in the PR description as part of your cleanups)?

Reply:
Forgot to mention it.  This attribute was unused, so there was no point to keep it while refactoring.